packages/ui-extensions/src/surfaces/admin/components/ChoiceList/ChoiceList.ts

```diff
@@ -0,0 +1,12 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import {InputProps} from '../shared';
+
+export interface ChoiceListProps
+  extends Pick<
+    InputProps<string | string[]>,
+    'value' | 'name' | 'onChange' | 'disabled' | 'error'
+  > {}
```

Comment:
I think we should add `defaultValue` and `readOnly` to `InputProps` and use them here

```typescript
/**
   * A `string` or `string[]` indicating the initial ids of selected choices
   * in an uncontrolled ChoiceList.
   * In case of `multiple: true` and an array use only the first value.
   */
  defaultValue?: T;
  
    /**
   * Whether the field is read-only.
   */
  readOnly?: boolean;
```

Reply:
And also `multiple` and `thisInstance`. I will add them